docker-compose/ext-src/h3-pg-src/neon-test.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

If you need bash, the correct shebang is ^